tools/cldr-code/src/test/java/org/unicode/cldr/util/TestVettingViewer.java

```diff
@@ -0,0 +1,90 @@
+package org.unicode.cldr.util;
+
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.EnumSet;
+import java.util.Map.Entry;
+
+import com.ibm.icu.impl.Relation;
+import com.ibm.icu.impl.Row.R2;
+
+import org.junit.jupiter.api.Test;
+import org.unicode.cldr.test.OutdatedPaths;
+import org.unicode.cldr.util.PathHeader.PageId;
+import org.unicode.cldr.util.PathHeader.SectionId;
+import org.unicode.cldr.util.VettingViewer.Choice;
+import org.unicode.cldr.util.VettingViewer.VoteStatus;
+
+/**
+ * Also see {@link org.unicode.cldr.unittest.TestUtilities}
+ */
+class TestVettingViewer {
+    static final String CENTURY_NARROW_DISPLAY_NAME = "//ldml/units/unitLength[@type=\"narrow\"]/unit[@type=\"duration-century\"]/displayName";
+    @Test
+    void testOutdatedPaths() {
+        String previous = VettingViewer.getOutdatedPaths().getPreviousEnglish(CENTURY_NARROW_DISPLAY_NAME);
+        assertNotEquals(OutdatedPaths.NO_VALUE, previous, "expected prev english to be present for " + CENTURY_NARROW_DISPLAY_NAME);
```

Comment:
@macchiati perhaps this is to be expected, though?

Reply:
It turns out that the English value was not explicitly set until v37 (aliased value), so it is predated in some locales.

Reply:
this particular test fails (for the reason you gave)  I can omit it, but keep the other test function.